docs/examples/hypervisors/kvm/main.tf

```diff
@@ -0,0 +1,85 @@
+provider "oci" {
+  tenancy_ocid     = "${var.tenancy_ocid}"
+  user_ocid        = "${var.user_ocid}"
+  fingerprint      = "${var.fingerprint}"
+  private_key_path = "${var.ssh_private_key_path}"
+  region           = "${var.region}"
+}
+
+resource "oci_core_instance" "KVM-HOST" {
+  availability_domain = "${lookup(data.oci_identity_availability_domains.ADs.availability_domains[var.availability_domain - 1],"name")}"
+  compartment_id      = "${var.compartment_ocid}"
+  display_name        = "KVM-HOST-${var.customer_name}"
+  image               = "${lookup(data.oci_core_images.BaseImage.images[0], "id")}"
+  shape               = "${var.instance_shape}"
+
+  create_vnic_details {
+    subnet_id        = "${oci_core_subnet.frontend-subnet.id}"
+    hostname_label   = "bmcs-host"
+    display_name     = "bmcs-host-vnic"
+    assign_public_ip = true
+  }
+
+  metadata {
+    ssh_authorized_keys = "${file(var.ssh_public_key_path)}"
+  }
+
+  timeouts {
+    create = "10m"
+  }
+}
+
+resource "oci_core_vnic_attachment" "kvm-mgmt-vnic-attachmnt" {
```

Comment:
Use "kvm-guest" prefix to clearly identify the VNICs created for the guest.
"kvm-mgmt-vnic-attachmnt" -> "kvm-guest-mgmt-vnic-attachmnt"
 "frontend-vnic-attachmnt" -> "kvm-guest-frontend-vnic-attachmnt"
"backend-vnic-attachmnt" -> "kvm-guest-backend-vnic-attachmnt"

Same thing for display_name
"kvm-mgmt-vnic" -> "kvm-guest-mgmt-vnic"
"kvm-frontend-vnic" -> "kvm-guest-frontend-vnic"
"kvm-backend-vnic" -> "kvm-guest-backend-vnic"